cmd/gke-gcloud-auth-plugin/main.go

```diff
@@ -42,28 +52,48 @@ var (
 	useAdcPtr = pflag.Bool("use_application_default_credentials", false, "returns exec credential filled with application default credentials.")
 )
 
+type pluginContext struct {
```

Comment:
Make the functions below be member functions of the struct.

Reply:
This just makes it easier to test. I ll follow up on this in the end.